rules/integrations/aws/discovery_ec2_multi_region_describe_instances.toml

```
@@ -107,6 +109,13 @@ from logs-aws.cloudtrail-*
 | sort target_time_window desc
 '''
 
+[rule.investigation_fields]
+field_names = [
+    "region_count",
+    "window_count",
+    "target_time_window",
+    "aws.cloudtrail.user_identity.arn"
```

```suggestion
    "aws.cloudtrail.user_identity.arn",
    "region_count",
    "window_count",
    "target_time_window"
```
If the order matters we should probably put the "who" first

---

See https://github.com/elastic/detection-rules/pull/4261#discussion_r1834863713